### PR TITLE
Steps to issue a credential (Manual flow)

### PR DESCRIPTION
**Pull request by Selbin**

### CODE REVIEW

**Comment by georgepadayatti** [comment]
@Selbin Update the commit message to `Steps to issue a credential (Manual flow)`

**Review by georgepadayatti — CHANGES_REQUESTED** [review]
Update the commit message

**Review by georgepadayatti — APPROVED** [review]

File: credential-issue-and-verification-api-user-guide.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ In your browser you can start three tabs to execute the APIs using swagger.
 | Alice (Data4Life User) | [agent2.swagger.localhost](http://agent2.swagger.localhost) | 
 | Travel Company | [agent3.swagger.localhost](http://agent3.swagger.localhost) | 
 
+# Work flow
```
Comment (lalc) [inline credential-issue-and-verification-api-user-guide.md:30]: super

**Review by lalc — APPROVED** [review]
Since the doc it long, can you consider putting a table of Contents?

**Review by lalc — APPROVED** [review]

**Pull request merged**